src/OrchardCore.Modules/OrchardCore.Users/Views/UserFields.cshtml

```diff
@@ -1,6 +1,16 @@
+@using Microsoft.AspNetCore.Identity
+
 @model SummaryAdminUserViewModel
 
+@inject UserManager<IUser> UserManager
+
 <h5 class="float-start">@Model.User.UserName <span class="hint dashed">@Model.User.Email</span></h5>
+@if (!await UserManager.IsEmailConfirmedAsync(Model.User))
+{
+    <span class="badge text-bg-warning ms-2" title="@T["Email not confirmed"]">
```

Comment:
```suggestion
    <span class="badge text-bg-warning ms-2" title="@T["The email address {0} has not been confirmed."]">
```

Reply:
Personally I like this language better.

If you want to keep what you have, at least you should start the sentence with the word "The confirmation...."

Reply:
Hehe, AI told me to remove the 'The' 🤭

Anyway, I fine with your suggestion but I think it should then match the text in the badge. So I would then suggest `Email not confirmed` instead of `Pending`. We can also go with your shorter `Not confirmed` label.

Reply:
I think I'll go with `Unconfirmed` in that case.

Reply:
What do you think:

![image](https://github.com/user-attachments/assets/c456d951-d440-42f7-9e6f-9a3f80a315f9)

Reply:
I like it as is now. Feel free to merge it.